platform/features/plot/src/PlotOptionsForm.js

```diff
@@ -0,0 +1,166 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(
+    [],
+    function () {
+        "use strict";
+
+        /**
+         * A class for encapsulating structure and behaviour of the plot
+         * options form
+         * @memberOf platform/features/plot
+         * @param topic
+         * @constructor
+         */
+        function PlotOptionsForm(topic) {
+            var self = this;
+            this.onchangeTopic = topic();
+
+            function onchange(value){
```

Comment:
It's a *bit* crazy to specify it against each control in the form spec for this particular case, where you're only interested in a single notification for any change in the form values, I agree. 

What I think is not crazy is having the __ability__ to have per-control change handling, and extending the form specification standard to support this.

The rationale behind this was that I wanted to avoid two-way data binding and additional watches, etc, and to implement a _somewhat_ angular agnostic approach. Granted, ng-change is an angular directive, and behaves differently to onchange, but it's similar enough that I think ng-change could be refactored out in future for an approach that uses native event handling, or the event handling of whatever view framework we end up using.

That being said, I'm open to the idea of extending mct-form to listen to changes in all its controls automatically, to allow for a single onchange/ng-change event handler to be registered.